Does not work with urllib3

It seems that pybotvac is not compatible with urllib3 >= 2

See https://github.com/home-assistant/core/pull/145485

```console
==================================== ERRORS ====================================
_________ ERROR collecting tests/components/neato/test_config_flow.py __________
tests/components/neato/test_config_flow.py:5: in <module>
    from pybotvac.neato import Neato
venv/lib/python3.13/site-packages/pybotvac/__init__.py:1: in <module>
    from .account import Account
venv/lib/python3.13/site-packages/pybotvac/account.py:22: in <module>
    from .robot import Robot
venv/lib/python3.13/site-packages/pybotvac/robot.py:25: in <module>
    urllib3.disable_warnings(urllib3.exceptions.SubjectAltNameWarning)
E   AttributeError: module 'urllib3.exceptions' has no attribute 'SubjectAltNameWarning'
=========================== short test summary info ============================
ERROR tests/components/neato/test_config_flow.py - AttributeError: module 'urllib3.exceptions' has no attribute 'SubjectAltNameWarning'
```

https://github.com/home-assistant/core/actions/runs/15203769428/job/42762619681

--- a/pybotvac/robot.py
+++ b/pybotvac/robot.py
@@ -5,7 +5,6 @@ from datetime import datetime, timezone
 from email.utils import format_datetime
 
 import requests
-import urllib3
 from voluptuous import (
     ALLOW_EXTRA,
     All,
@@ -19,10 +18,6 @@ from voluptuous import (
 
 from .exceptions import NeatoRobotException, NeatoUnsupportedDevice
 from .neato import Neato  # For default Vendor argument
-
-# Disable warning due to SubjectAltNameWarning in certificate
-# pylint: disable=no-member
-urllib3.disable_warnings(urllib3.exceptions.SubjectAltNameWarning)
 
 _LOGGER = logging.getLogger(__name__)
 

